dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/association/ProgramOrganisationUnitAssociationsQueryBuilder.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (c) 2004-2004-2020, University of Oslo
```

Comment:
```suggestion
 * Copyright (c) 2004-2021, University of Oslo
```

Reply:
we removed the old implementation.
@ameenhere commented on this below, I think